main.py

```diff
@@ -2,12 +2,30 @@
 import pyautogui
 import sys
 import uvicorn
-from fastapi import FastAPI
+from fastapi import FastAPI, WebSocket
 from fastapi.staticfiles import StaticFiles
 from fastapi.responses import FileResponse
 
 app = FastAPI()
+pyautogui.FAILSAFE = False
+pyautogui.PAUSE = 0
```

Comment:
What is the default value? Let's move pyautogui setup lines to `if __name__ == "__main__"` block.